packages/generator/templates/ClientInstance.ts.hbs

```diff
@@ -21,26 +15,26 @@ import {
 * operations it is used to perform.
 */
 class {{capitalize apiSpec}} extends BaseClient {
-  constructor(config: ClientConfig) {
-    super(config);
+constructor(config: ClientConfig) {
+super(config);
 
-    if (!!!config.baseUri) {
-      this.clientConfig.baseUri = "{{getBaseUri apiModel}}";
-    }
+if (!!!config.baseUri) {
```

Comment:
I know you didn't write this, but why is there a `!!!` here? Shouldn't `!` be sufficient?

Reply:
I wrote it. I'm not sure why it's there.